Symfony/CS/Tests/Console/TextDiffTestInput.test

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I tried the fixture, but if you run this unit test on master from the root directory of the project the file gets excluded because of the intersection made based on the configuration file in the root directory (it was also excluded because it ended with `.test` I think).
I think that was why I moved it here, but indeed it is not needed. I'll refactor soon and process your other points as well. Thanks for reviewing :+1: